build_tools/ci/run_matmul_test.sh

```diff
@@ -39,6 +56,13 @@ TEST_RUNNER="${IREE_INSTALL_BIN}/iree-e2e-matmul-test"
 echo "Python version: $("${IREE_PYTHON3_EXECUTABLE}" --version)"
 echo "iree-compile version: $("${IREE_COMPILE_EXE}" --version)"
 mkdir -p ${OUTPUT_DIR}
+cd ${OUTPUT_DIR}
+
+python3 -m venv sandbox
```

Comment:
Good point - could install deps once then let the test steps reuse that same environment.

Reply:
Can you link to some logs?

Reply:
https://github.com/nod-ai/iree-amd-aie/actions/runs/7934375875/job/21665336705?pr=170

I am suspecting this is failing because the MLIR_AIE_INSTALL path is not correct which I get from the venv

Reply:
Here is a successful one before venv sharing
https://github.com/nod-ai/iree-amd-aie/actions/runs/7933813811/job/21663559947

Reply:
Bah GitHub Actions troubleshooting... Maybe try `python -m pip install` instead of `pip install`, to make sure  you use pip from the venv?

Reply:
Thanks, just had to find the path to the venv correctly.